mautic/mautic
# Page bounce query optimization

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | N
| New feature? | N
| Related user documentation PR URL | N
| Related developer documentation PR URL | N
| Issues addressed (#s or URLs) | N
| BC breaks? | N
| Deprecations? | N

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
This PR optimizes the page bounce query times on pages with many hits. Drops the creation on of a temp table so as to not bog down MySQL when working with millions of rows. It also includes @escopecz's fix #2372  for removing the bounces from the page load view, since it's not used there anyway.

#### Steps to test this PR:
1. Apply PR
2. Test to ensure that page detail views are loading. If you have a lot of page hits registered, it should load faster.
